### PR TITLE
Server 1978 azure registry cmds

### PR DESCRIPTION
**Pull request by soulchips**
:gear: **Issue**

updates to the migration script's instructions in the output
flyway migration image will now pull from azure cciserver
added --dev flag tp allow users to pull from azure dev chart during testing

:white_check_mark: **Fix**

:question: **Tests**

- [ ] Tested Updating Existing Instance
- [ ] Installed on new instance

### CODE REVIEW

File: kots-exporter/kots-exporter.sh [hunk 1]
```diff
@@ -120,6 +120,17 @@ set_default_value(){
...
+    # check if this is a dev migration
+    if [[ $dev == true ]]
+    then
+        export REGISTRY="devcharts"
+        echo "Using dev azure registry"
+    else
+        export REGISTRY="cciserver"
+    fi
+
+    export CHART="oci://devcharts.azurecr.io/circleci-server"
```
Comment (ryanwohara) [inline kots-exporter/kots-exporter.sh:133]: This line should use the $REGISTRY value instead of hardcoding devcharts

Comment (soulchips) [reply kots-exporter/kots-exporter.sh:133]: aah thanks. i missed that. good call out!

**Review by atulsingh0 — APPROVED** [review]

File: kots-exporter/templates/circle-migrator.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ spec:
       - name: regcred
       containers:
       - name: circle-migrator
-        image: circleci/circle-migrator:0.1.532114-6a5071e0ed
+        image: "$REGISTRY".azurecr.io/circle-migrator:0.1.532114-6a5071e0ed
```
Comment (atulsingh0) [inline kots-exporter/templates/circle-migrator.yaml:18]:
```suggestion
        image: $REGISTRY.azurecr.io/circle-migrator:0.1.532114-6a5071e0ed
```

**Review by lloydChris — APPROVED** [review]
nit: I might have passed the registry name in, instead of the dev toggle, but potato potato

File: kots-exporter/templates/circle-migrator.yaml (hunk 2 above)
Comment (soulchips) [reply kots-exporter/templates/circle-migrator.yaml:18]: good catch!

**Review by atulsingh0 — APPROVED** [review]

**Pull request merged**